src/main/java/com/enterprise/eventmanagementsystem/service/EventService.java

```diff
@@ -42,15 +44,17 @@ public Event save(Event event) throws Exception {
     }
 
```

Comment:
I added caching as it is good coding practice the increases the performance of the website greatly.